tests/osaka/eip7692_eof_v1/eip7620_eof_create/test_eofcreate.py

```diff
@@ -282,7 +282,7 @@ def test_calldata(state_test: StateTestFiller, pre: Alloc):
             sections=[
                 Section.Code(
                     code=Op.MSTORE(0, Op.PUSH32(calldata))
-                    + Op.SSTORE(slot_create_address, Op.EOFCREATE[0](0, 0, 0, calldata_size))
+                    + Op.SSTORE(slot_create_address, Op.EOFCREATE[0](input_size=calldata_size))
```

Comment:
All of these changes could go in even if the opcode stack order doesn't change in the end 😀 

Is there any other opcode getting a similar change? We could prepare a PR with only keyword-argument changes.

Reply:
I have not gotten to doing the split, but at this point we're ready to merge the eof-devnet-1 changes (which this PR is part of), so I think we can move ahead as is